docs/quickstart.md

```diff
@@ -172,95 +181,116 @@ If the KubeAdmiral control plane is working properly, the deployment will be pro
 $ kubectl get deploy echo-server
 
 NAME          READY   UP-TO-DATE   AVAILABLE   AGE
-echo-server   6/6     6            6           10m
+echo-server   6/6     6            6           26s
 ```
 
-We can also view the status of the deployments in each member cluster by checking its `FederatedDeploymentStatus` object.
+We can also view the status of the deployments in each member cluster by checking its `CollectedStatus` object.
 
 ```console
-$ kubectl get fdeploystatus echo-server -oyaml
+# Usually the name consists of the resource name and its ResourcesGroup,
```

Comment:
```suggestion
# Usually the name consists of the resource name and its API group (e.g. echo-server-deployments.apps), 
```

Reply:
This is a `console` block, where lines starting with `#` and `$` are treated as command lines (for root and non-root users respectively). `#` does not represent comments here.